content/applications/general/email_communication/email_domain.rst

```diff
@@ -0,0 +1,162 @@
+called *chatter*.
+
+When you post a message in the chatter, this message is sent by email to the followers of the
+document. If a follower replies to the message, the reply updates the chatter, and Odoo relays the
+reply to the followers.
+
+Emails from your users to partners (customers, vendors) are sent from the email address of your
+users. Similarly, emails from partners to users are sent from the email address of the partners.
+This allows you to recognize at a glance who sent an email relayed by Odoo.
+
+If your database is hosted on our cloud (Odoo Online or Odoo.sh), it is not necessary to add an
+outgoing email server to send emails from your custom domain. You can enjoy this feature by using
+the default Odoo email server.
+
+.. important:: The Odoo server is subject to a daily email limit to prevent abuse. The default
+   limit is 200 emails sent per day for databases with an Enterprise subscription. This limit can be
+   increased under certain conditions. See our `FAQ <./email_common.html>`_ or contact support for
+   more information.
+
+However, it is recommended that you configure your domain name to ensure that emails from your
+users reach your partners, rather than being considered spam.
+
+For the same reason, we recommend that you always give your users an email address from a domain
+you manage, rather than a generic email address (gmail.com, outlook.com, etc.).
+
+.. _spf_compliant:
+
+Be SPF compliant
+================
+
+The Sender Policy Framework (SPF) protocol allows the owner of a domain name to specify which
+servers are allowed to send email from that domain.
+
+When a server receives an incoming email, it checks whether the IP address of the sending server is
+on the list of allowed IPs according to the SPF record of the sender.
+
+.. note:: The SPF verification is performed on the domain mentioned in the Return-Path field of the
+   email. In the case of an email sent by Odoo, this domain corresponds to the value of the 
+   `mail.catchall.domain` key in the database system parameters.
+
+   See the `documentation <https://www.odoo.com/documentation/14.0/applications/productivity/discuss/advanced/email_servers.html#how-to-manage-inbound-messages>`_ on incoming email.
+
+The SPF policy of a domain is set using a TXT record.
+
+How to create or modify a TXT record depends on the provider hosting the DNS zone of your domain
+name.
+
+In order for the verification to work properly, each domain can only have one SPF record.
+
+If your domain name does not yet have an SPF record, the content of the record to create is as
+follows:
+
+``v=spf1 include:_spf.odoo.com ~all``
+
+If your domain name already has an SPF record, you need to update this record (and do not create a
+new one).
+
+For example, if your TXT record is:
+
+``v=spf1 include:_spf.google.com ~all``
+
+You need to edit it to add ```include:_spf.odoo.com``:
+
+``v=spf1 include:_spf.odoo.com include:_spf.google.com ~all``
```

Comment:
I used the "tip" one however I don't think it's the most relevant for example.
Here is the list I found on the choice I could make: "attention", "caution", "danger", "error", "hint", "important", "note", "tip", "warning", "admonition" 

But "admonition" is not usable 

Advice welcomed :) otherwise I leave it as "tip"